web/api/v1/api.go

```diff
@@ -783,6 +792,14 @@ func (api *API) labelValues(r *http.Request) (result apiFuncResult) {
 
 	slices.Sort(vals)
 
+	limit, err := parseLimitParam(r.FormValue("limit"))
```

Comment:
Can we move this before we actually fetch data? So that we don't throw a 400 error after we finish fetching label values